filament/backend/src/opengl/OpenGLContext.h

```diff
@@ -429,14 +428,6 @@ void OpenGLContext::bindVertexArray(RenderPrimitive const* p) noexcept {
     RenderPrimitive* vao = p ? const_cast<RenderPrimitive *>(p) : &mDefaultVAO;
     update_state(state.vao.p, vao, [&]() {
         glBindVertexArray(vao->vao);
-        // update GL_ELEMENT_ARRAY_BUFFER, which is updated by glBindVertexArray
-        size_t targetIndex = getIndexForBufferTarget(GL_ELEMENT_ARRAY_BUFFER);
-        state.buffers.genericBinding[targetIndex] = vao->elementArray;
```

Comment:
Similar question, even though we don't skip a bind operation based on a cached value anymore, `glBindVertexArray()` here still modifies the state at `GL_ELEMENT_ARRAY_BUFFER` target. Thus, `state.buffers.genericBinding[targetIndex]` needs an update, right?

Reply:
As discussed: [9c36b58](https://github.com/shapr3d/filament/pull/94/commits/9c36b5867b132b2bfc2471332ced52acea51e74f)